.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,45 @@
+name: ✨ Feature Request
+description: Request a new feature
+labels: ["enhancement", "needs-triage"]
+# assignees:
+#   - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there 👋,
+
+        Thank you for requesting a new feature!
+  - type: checkboxes
+    id: type
+    attributes:
+      label: Feature Type
+      description: What do you want to request?
+      options:
+        - label: a new resource for an upstream API endpoint
+          required: false
+        - label: a new datasource for an upstream API endpoint
+          required: false
+        - label: align attributes with an upstream API endpoint
+          required: false
+        - label: something else
+          required: false
```

Comment:
I think cut this field. It uses language that we're familiar with as maintainers but might be foreign to the user.

```suggestion
```